### DISCUSSION TITLE
Can i set prPriority without grouping packages?

### DISCUSSION THREAD

**Discussion by waszakCeneo** [renovatebot/renovate#23705] [Request Help] [answered]
### How are you running Renovate?

Placeholder value, please select the correct response from the dropdown

### If you're self-hosting Renovate, tell us what version of Renovate you run.

36.29.1

### If you're self-hosting Renovate, select which platform you are using.

Azure DevOps (dev.azure.com)

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I want to have the same PR priority for packages starting with prefix "ABC."
but i don't want to group them in the same package rule

### What you tried so far.

I tried this but this would group all changes in one PR. And i want multiple PRs with higher prirority
`  {
      "groupName": "Abc",
      "matchPackagePrefixes": [
        "Abc"
      ],
      "prPriority": 5,
      "rangeStrategy": "bump"
    }`

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>
/
/

**Comment by rarkins** [comment, marked as answer]
Yes you can, you don't need to set `groupName`

**Comment by waszakCeneo** [comment]
Thanks for help
